tests/integration/test_integration.py

```diff
@@ -535,6 +806,308 @@ def test_uniques_subset_within(engine, unique_table1, data):
+            0,
+            None,
+            False,
+            output_postprocessing_sorter,
+            slice(None),
+            None,
+            None,
+            "column(s) 'col_int' has a fraction of 0.03333333333333333 > 0 values (2 / 60) not being an element of '[0, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29]'. It has e.g. (slice(None, None, None)) excess elements '[3]' with counts [2].",
+        ),
+        (
+            negation,
+            ["col_int"],
+            [i for i in range(30) if i != 3],
+            0,
+            None,
+            False,
+            output_postprocessing_sorter,
+            slice(None),
+            None,
+            None,
+            "column(s) 'col_int' has a fraction of 0.03333333333333333 > 0 values (2 / 60) not being an element of '[0, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29]'. It has e.g. (slice(None, None, None)) excess elements '[3]' with counts [2].",
+        ),
+        (
+            negation,
+            ["col_int"],
+            [i for i in range(30) if i != 3],
+            0,
+            None,
+            True,
+            output_postprocessing_sorter,
+            slice(None),
+            None,
+            None,
+            "column(s) 'col_int' has a fraction of 0.03333333333333333 > 0 DISTINCT values (1 / 30) not being an element of '[0, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29]'. It has e.g. (slice(None, None, None)) excess elements '[3]' with counts [2].",
+        ),
+        (
+            negation,
+            ["col_int"],
+            [i for i in range(30) if i != 3],
+            0,
+            filternull_element,
+            False,
+            output_postprocessing_sorter,
+            slice(None),
+            None,
+            None,
+            "column(s) 'col_int' has a fraction of 0.03333333333333333 > 0 values (2 / 60) not being an element of '[0, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29]'. It has e.g. (slice(None, None, None)) excess elements '[3]' with counts [2].",
+        ),
+        (
+            negation,
+            ["col_int"],
+            [i for i in range(30) if i != 3],
+            0,
+            filternull_element_or_tuple_all,
+            False,
+            output_postprocessing_sorter,
+            slice(None),
+            None,
+            None,
+            "column(s) 'col_int' has a fraction of 0.03333333333333333 > 0 values (2 / 60) not being an element of '[0, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29]'. It has e.g. (slice(None, None, None)) excess elements '[3]' with counts [2].",
+        ),
+        (
+            negation,
+            ["col_int"],
+            [i for i in range(30) if i != 3],
```

Comment:
Another reason why this is better, now I explicitly see that the number 3 was missing from the previous list!